contracts/scripts/start-hardhat.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+set -meuo pipefail
+
+function cleanup {
+  jobs -p | xargs kill
+}
+
+trap cleanup EXIT
+
+yarn hardhat node &
+
+sleep 2
```

Comment:
Node spinup may take longer on older/slower machines.

```suggestion
sleep 5
```

Reply:
`sleep 2` was already a lot slower than was needed in my testing. Any particular reason you feel 5 seconds is the right number?